builder/bin/image_gen.rs

```diff
@@ -41,6 +49,18 @@ fn main() {
         std::fs::write(path, rom).unwrap();
     }
 
+    let fmc_svn = if let Some(fmc) = args.get_one::<u32>("fmc-svn") {
```

Comment:
optional: what do you think of using `or_default` on these to be more concise?

Reply:
Going to leave this because `get_one::<u32>` returns `Option<&u32>` and Default isn't implemented for `&u32`.